index.md

```diff
@@ -20,25 +20,58 @@ permalink: /
 
-Learning a new specification can be daunting.
+Learn, Get help, Shape the Community, Chat, with the JSON Schema team and Community!
 
-You should read our [getting started guide](/learn/getting-started-step-by-step)!
+<div class="wrapper text-center buttons">
+    <a class="button border button-center" href="/learn/getting-started-step-by-step">👋 Getting Started</a>
+    <a class="button border button-center" href="/slack" target="_blank"><img class="small-svg-logo" src="/assets/logo-slack.svg" height="1.3em" width="1.3em"> Open JSON Schema Slack</a>
+    <a class="button border button-center" href="https://github.com/json-schema-org/community/discussions" target="_blank">💬 Community Discussions</a>
+</div>
+
+## Regular Activities
+
+We hold weekly Office Hours and twice monthly Open Community Working Meetings.
+
+<div class="wrapper text-center buttons">
+    <a class="button border button-center" href="https://github.com/json-schema-org/community/discussions/34" target="_blank">🧑‍💻 Office Hours</a>
+    <a class="button border button-center" href="https://github.com/json-schema-org/community/discussions/35" target="_blank">👷 Open Community Working Meetings</a>
+</div>
+
+Office Hours are every Tuesday at 15:00 UTC.
+
+Open Community Working Meetings follow two patterns:
+- First Friday of the month at 20:00 UTC.
+- Third Friday of the month at 15:00 UTC.
+
+## Need more?
+
+We have our other [learning resources](/learn), including the [Understanding JSON Schema documentation](/understanding-json-schema).
+
+## About Our Community
 
-You can also see our other [learning resources](/learn).
+We have an active and growing community. All are welcome to be part of our community, help shape it, or simply observe.
 
-### Got questions?
+We want keep our community welcoming and inclusive, so please read our [JSON Schema Organizational Code of Conduct](https://github.com/json-schema-org/.github/blob/main/CODE_OF_CONDUCT.md). (This is a combination of the Contributor Covenant and IETF BCP 54.)
 
 The JSON Schema team and community are here to help!
 
-At any point, feel free to join our [Slack server](/slack).
+At any point, feel free to join our Slack server. [Slack server](/slack).
 
-We also monitor the `jsonschema` tag on StackOverflow.
+Our Slack server has limited history, so we also use [GitHub Discussions](https://github.com/json-schema-org/community/discussions).
+
+We monitor the `jsonschema` tag on StackOverflow.
 
 ## Project Status
 
-2021-02-01: Draft 2020-12 has been published!
+We have a twice yearly release cycle for the specification.
+
+We are currently in the "patch release" cycle which is tracked by [Milestone 9](https://github.com/json-schema-org/json-schema-spec/milestone/9).
+
+One release includes new features ("feature release"), while the other release includes non-controversial clarifications and corrections that do not change compliance requirements ("patch release"). These should be spaced roughly 6 months apart.
```

Comment:
I think saying we release twice a year could be misleading. I think we should say that we aim for a major release once a year. Then we can say that we also try to do a patch update mid-cycle if needed. I wouldn't call the patch a "release" because there's nothing new being introduced. Implementations don't need to update their code and schema authors don't need to update their schemas.